pkg/vmimage/data/script.sh

```diff
@@ -174,7 +204,11 @@ EOF
 # not commited with a : so that Windows users can check out the repo
 mv /etc/docker/certs.d/docker-registry.default.svc-5000 /etc/docker/certs.d/docker-registry.default.svc:5000
 
-rm /var/lib/yum/client-cert.pem /var/lib/yum/client-key.pem
+rm -fv /etc/yum.repos.d/kickstart.repo \
```

Comment:
Previously we did not remove the kickstart repo, why are we doing that now?

Reply:
must not remove that - see https://github.com/openshift/openshift-azure/blob/93a30816ad15aa87636a608fe7b0233023c3af7b/pkg/arm/v7/data/master-startup.sh#L18